### PR TITLE
Parametrize MIN_CAN_SPEED in car interfaces

### PR DESCRIPTION
**Pull request by briskspirit**
Saw TODO inside longcontrol.py to parametrize MIN_CAN_SPEED after I had to fix this value for 2021 Corolla TSS2 (MIN_CAN_SPEED should be 0.375
Base value was added to each car make interface.py to get_params as ret.minCANSpeed = 0.3

### CODE REVIEW

File: selfdrive/car/volkswagen/interface.py [hunk 1]
```diff
@@ -78,7 +81,7 @@ def update(self, c, can_strings):
     # TODO: add a field for this to carState, car interface code shouldn't write params
     # Update the device metric configuration to match the car at first startup,
     # or if there's been a change.
-    #if self.CS.displayMetricUnits != self.displayMetricUnitsPrev:
+    # if self.CS.displayMetricUnits != self.displayMetricUnitsPrev:
```
Comment (adeebshihadeh) [inline selfdrive/car/volkswagen/interface.py:84]: revert this

**Comment by briskspirit** [comment]
Done

File: selfdrive/car/honda/interface.py [hunk 2]
```diff
@@ -29,16 +30,16 @@ def get_compute_gb_acura():
...
+                 [1.03691769, 0.78210306, -0.41343188]])
+  b0 = np.array([0.01536703, -0.14335321, -0.26932889])
+  w2 = np.array([[-0.59124422, 0.42899439, 0.38660881],
+                 [0.79973811, 0.13178682, 0.08550351],
+                 [-0.15651935, -0.44360259, 0.76910877]])
+  b2 = np.array([0.15624429, 0.02294923, -0.0341086])
   w4 = np.array([[-0.31521443],
                  [-0.38626176],
-                 [ 0.52667892]])
+                 [0.52667892]])
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/interface.py:42]: revert all of these changes

File: selfdrive/car/gm/interface.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 from cereal import car
 from selfdrive.config import Conversions as CV
 from selfdrive.car.gm.values import CAR, CruiseButtons, \
-                                    AccState
+    AccState
```
Comment (adeebshihadeh) [inline selfdrive/car/gm/interface.py:5]: revert this

**Comment by briskspirit** [comment]
Switched off PEP8 autoformat, sorry... Reverted.

**Pull request merged**